README.md

```diff
@@ -98,7 +98,30 @@ RAILS_ENV=production bundle exec rake drop[services-disk01]
 RAILS_ENV=production bundle exec rake populate[services-disk01]
 ```
 
+### Run Moab to Catalog existence check for a single root and for all storage roots
 
+#### Single Root
+- Without profiling
+```ruby
+bundle exec rake m2c_exist_single_root[fixture_sr1,false]
```

Comment:
can you say something about 'false' ?  ("Where the boolean is for "expect_to_create" -- that is, if the object is not in the catalog, should we add it now?")

TBH, I don't think this functionality was included in confirm_version, and I think it is going to be included in check_existence ... so I wonder if we still want the argument at all.

Moreover, if we DO want the boolean arg, we would also want it for the all roots version ...    perhaps another question for storytime?